### PR TITLE
Use GetZoneInfo to determine domain features

### PR DESCRIPTION
**Pull request by h9k**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,11 +40,17 @@
   ],
   "devDependencies": {
     "@hexonet/semantic-release-github-whmcs-config": "^1.3.1",
+    "cz-conventional-changelog": "^3.3.0",
```
Comment (papakai) [inline package.json:43]: not necessary

commitizen is installed globally. I never had the requirement to specify cz-conventional-changelog explicitely.
Can you provide more insights why you are forwading this? I just know these changes are part of the github readme of commitizen, right? If you had no explicit (technical) reason, I suggest not doing this.

File: package.json [hunk 2]
```diff
@@ -40,11 +40,17 @@
...
     "semantic-release": "^17.1.1"
   },
   "scripts": {
     "codeCheck": "phpcs --standard=PSR12 -q -n --colors modules",
     "codeFix": "phpcbf --standard=PSR12 -q modules",
     "phpCompatibility": "./scripts/phpcompatibility.sh"
+  },
+  "config": {
```
Comment (papakai) [inline package.json:51]: not necessary, see previous comment

**Review by papakai — CHANGES_REQUESTED** [review]

**Comment by papakai** [comment]
When the above is resolved, feel free to directly merge. ty!

File: package.json (hunk 1 above)
Comment (h9k) [reply package.json:43]: I had the issue where `git cz` would again just open the normal commit vi, so I ran that initialization command `commitizen init cz-conventional-changelog --save-dev --save-exact` again, and that added the stuff to package.json.

Comment (papakai) [reply package.json:43]: hm, i checked something:
`npm i -g cz-conventional-changelog` ftw!

Please just delete the commit introducing the package.json changes. When having the above module installed globally, it does what you need.

Comment (h9k) [reply package.json]: Allright! Done.

**Pull request merged**